Filters/exceptions.txt

```diff
@@ -3,6 +3,9 @@
 ! `*-dnsotls-ds.metric.gstatic.com` (for instance, `a5a6380f-dnsotls-ds.metric.gstatic.com`).
 !
 !
+! https://github.com/AdguardTeam/AdGuardDNS/issues/574
```

Comment:
It seems that there is incorrect link, it should be:
`https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/574`

Reply:
Changed.